rules/windows/process_creation/proc_creation_win_regedit_export_critical_keys.yml

```
@@ -10,29 +10,31 @@ references:
     - https://gist.github.com/api0cradle/cdd2d0d0ec9abb686f0e89306e277b8f
 author: Oddvar Moe, Sander Wiebing, oscd.community
 date: 2020/10/12
-modified: 2022/07/07
+modified: 2023/02/03
```

```suggestion
modified: 2022/07/07
```
we shouldn't change the modified date for cosmetic changes